keras_cv/models/clip/clip_tokenizer.py

```diff
@@ -0,0 +1,147 @@
+# encodes text
+import gzip
+import html
+import os
+from functools import lru_cache
+
+import ftfy
+import regex as re
+
+
+@lru_cache()
+def default_bpe():
+    return os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "bpe_simple_vocab_16e6.txt.gz"
+    )
+
+
+class CLIPTokenizer(object):
```

Comment:
I think this if no where else is an excellent opportunity to import from KerasNLP. We already have a BPE tokenizer and downstream GPT2 tokenizer!

Reply:
Agreed! I'll rely on those then :)
Thanks for the heads up!